integration/presto/pom.xml

```diff
@@ -462,12 +462,6 @@
       <version>3.0.2</version>
```

Comment:
Please check whether can remove the below hadoop dependency in presto/pom.xml, or not ?

<dependency>
      <groupId>com.facebook.presto.hadoop</groupId>
      <artifactId>hadoop-apache2</artifactId>
      <version>2.7.3-1</version>
      <exclusions>
        <exclusion>
          <groupId>org.antlr</groupId>
          <artifactId>antlr4-runtime</artifactId>
        </exclusion>
        <exclusion>
          <groupId>com.fasterxml.jackson.core</groupId>
          <artifactId>jackson-databind</artifactId>
        </exclusion>
      </exclusions>
    </dependency>

Reply:
We can not remove this class for now as compilation fails , is there a particular reason we should have not this dependency then I can look into supplementing it with some carbon jar

Reply:
Because in presto-module already have hadoop dependency as below, can we remove one ?
```
groupId>org.apache.carbondata</groupId>
<artifactId>carbondata-hadoop</artifactId>
<version>${project.version}</version>
```

Reply:
@chenliang613  we can not remove this dependency as CarbonTableInputFormat is part of this , we have excluded a lot of dependencies which were not needed in this particular dependency.